panic when trying to use anchors

I used this config with version v3.0.0-beta.2 to generate both mock and stub:
```yaml
template: matryer

# configs presets
anchors: &make_both_mock_and_stub
  configs:
  - template-data:
      stub-impl: false
    filename: mocks_test_exported.go
    structname: "Mock{{.InterfaceName}}"
  - template-data:
      stub-impl: true
    filename: mocks_test_exported.go
    structname: "Stub{{.InterfaceName}}"

# specify an explicit list of mocked interfaces
all: False
packages:
  path/to/repo:
    interfaces:
      MyInterface: *make_both_mock_and_stub
```

I updated to v3.3.1 recently, but received an error:
`FTL app failed error="getting config: unmarshalling config: decoding failed due to the following error(s):\n\n'' has invalid keys: anchors" version=v3.3.1`

I changed `anchors` to `_anchors` to fix an error, but i received panic:
```text
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/knadh/koanf/maps.Merge(0xa92a60?, 0x0)
        /home/sam/go/pkg/mod/github.com/knadh/koanf/maps@v0.1.1/maps.go:114 +0x1bc
github.com/knadh/koanf/maps.Merge(0xc000251290?, 0xc000250ed0)
        /home/sam/go/pkg/mod/github.com/knadh/koanf/maps@v0.1.1/maps.go:127 +0xf3
github.com/knadh/koanf/v2.(*Koanf).merge(0xc000250f60, 0xc000251290, 0xc00008c3c8)
        /home/sam/go/pkg/mod/github.com/knadh/koanf/v2@v2.1.2/koanf.go:416 +0x6b
github.com/knadh/koanf/v2.(*Koanf).Load(0xc000250f60, {0xc7a8e0?, 0xc0000a90e0?}, {0xc7a7f0?, 0x1183760?}, {0x0, 0x0, 0x0?})
        /home/sam/go/pkg/mod/github.com/knadh/koanf/v2@v2.1.2/koanf.go:120 +0x12e
github.com/vektra/mockery/v3/config.NewRootConfig({0xc7e618, 0xc000250e70}, 0xc000248200)
        /home/sam/go/pkg/mod/github.com/vektra/mockery/v3@v3.3.1/config/config.go:189 +0x745
github.com/vektra/mockery/v3/internal/cmd.GetRootApp({0xc7e618, 0xc000250e70}, 0xc000248200)
        /home/sam/go/pkg/mod/github.com/vektra/mockery/v3@v3.3.1/internal/cmd/mockery.go:84 +0x46
github.com/vektra/mockery/v3/internal/cmd.NewRootCmd.func1(0xc000248900?, {0xc0000a8e60?, 0x4?, 0xb2f609?})
        /home/sam/go/pkg/mod/github.com/vektra/mockery/v3@v3.3.1/internal/cmd/mockery.go:42 +0x206
github.com/spf13/cobra.(*Command).execute(0xc000193508, {0xc0000224f0, 0x2, 0x2})
        /home/sam/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:989 +0xabb
github.com/spf13/cobra.(*Command).ExecuteC(0xc000193508)
        /home/sam/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x44f
github.com/spf13/cobra.(*Command).Execute(...)
        /home/sam/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/vektra/mockery/v3/internal/cmd.Execute()
        /home/sam/go/pkg/mod/github.com/vektra/mockery/v3@v3.3.1/internal/cmd/mockery.go:73 +0x31
main.main()
        /home/sam/go/pkg/mod/github.com/vektra/mockery/v3@v3.3.1/main.go:8 +0xf
exit status 2
```

--- a/.mockery_testify.yml
+++ b/.mockery_testify.yml
@@ -1,3 +1,5 @@
+_anchors:
+  foo: bar
 template: "testify"
 force-file-write: true
 formatter: "goimports"

--- a/config/config.go
+++ b/config/config.go
@@ -68,6 +68,7 @@ func addr[T any](v T) *T {
 func NewDefaultKoanf(ctx context.Context) (*koanf.Koanf, error) {
 	c := Config{
 		All:                         addr(false),
+		Anchors:                     map[string]any{},
 		Dir:                         addr("{{.InterfaceDir}}"),
 		FileName:                    addr("mocks_test.go"),
 		ForceFileWrite:              addr(true),

--- a/go.work.sum
+++ b/go.work.sum
@@ -822,8 +822,6 @@ github.com/census-instrumentation/opencensus-proto v0.4.1/go.mod h1:4T9NM4+4Vw91
 github.com/cespare/xxhash v1.1.0 h1:a6HrQnmkObjyL+Gs60czilIUGqrzKutQD6XZog3p+ko=
 github.com/cespare/xxhash v1.1.0/go.mod h1:XrSqR1VqqWfGrhpAt58auRo0WTKS1nRRg3ghfAqPWnc=
 github.com/cespare/xxhash/v2 v2.2.0/go.mod h1:VGX0DQ3Q6kWi7AoAeZDth3/j3BFtOZR5XLFGgcrjCOs=
-github.com/cespare/xxhash/v2 v2.3.0 h1:UL815xU9SqsFlibzuggzjXhog7bL6oX9BbNZnL2UFvs=
-github.com/cespare/xxhash/v2 v2.3.0/go.mod h1:VGX0DQ3Q6kWi7AoAeZDth3/j3BFtOZR5XLFGgcrjCOs=
 github.com/chigopher/pathlib v1.0.0 h1:SbsCrFX4vDf4M2d8mT/RTzuVlKOjTKoPHK0HidsQFak=
 github.com/chigopher/pathlib v1.0.0/go.mod h1:3+YPPV21mU9vyw8Mjp+F33CyCfE6iOzinpiqBcccv7I=
 github.com/chzyer/logex v1.1.10 h1:Swpa1K6QvQznwJRcfTfQJmTE72DqScAa40E+fbHEXEE=
@@ -1043,6 +1041,7 @@ github.com/hashicorp/go-uuid v1.0.3 h1:2gKiV6YVmrJ1i2CKKa9obLvRieoRGviZFL26PcT/C
 github.com/hashicorp/go-uuid v1.0.3/go.mod h1:6SBZvOh/SIDV7/2o3Jml5SYk/TvGqwFJ/bN7x4byOro=
 github.com/hashicorp/golang-lru v0.5.4 h1:YDjusn29QI/Das2iO9M0BHnIbxPeyuCHsjMW+lJfyTc=
 github.com/hashicorp/golang-lru v0.5.4/go.mod h1:iADmTwqILo4mZ8BN3D2Q6+9jd8WM5uGBxy+E8yxSoD4=
+github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
 github.com/hashicorp/logutils v1.0.0 h1:dLEQVugN8vlakKOUE3ihGLTZJRB4j+M2cdTm/ORI65Y=
 github.com/hashicorp/logutils v1.0.0/go.mod h1:QIAnNjmIWmVIIkWDTG1z5v++HQmx9WQRO+LraFDTW64=
 github.com/hashicorp/mdns v1.0.4 h1:sY0CMhFmjIPDMlTB+HfymFHCaYLhgifZ0QhjaYKD/UQ=
@@ -1096,6 +1095,7 @@ github.com/lyft/protoc-gen-star/v2 v2.0.3 h1:/3+/2sWyXeMLzKd1bX+ixWKgEMsULrIivpD
 github.com/lyft/protoc-gen-star/v2 v2.0.3/go.mod h1:amey7yeodaJhXSbf/TlLvWiqQfLOSpEk//mLlc+axEk=
 github.com/magefile/mage v1.14.0 h1:6QDX3g6z1YvJ4olPhT1wksUcSa/V0a1B+pJb73fBjyo=
 github.com/magefile/mage v1.14.0/go.mod h1:z5UZb/iS3GoOSn0JgWuiw7dxlurVYTu+/jHXqQg881A=
+github.com/magiconair/properties v1.8.6/go.mod h1:y3VJvCyxH9uVvJTWEGAELF3aiYNyPKd5NZ3oSwXrF60=
 github.com/mattn/go-colorable v0.0.9/go.mod h1:9vuHe8Xs5qXnSaW/c/ABM9alt+Vo+STaOChaDxuIBZU=
 github.com/mattn/go-colorable v0.1.4/go.mod h1:U0ppj6V5qS13XJ6of8GYAs25YV2eR4EVcfRqFIhoBtE=
 github.com/mattn/go-colorable v0.1.6/go.mod h1:u6P/XSegPjTcexA+o6vUJrdnUu04hMope9wVRipJSqc=
@@ -1125,6 +1125,7 @@ github.com/mitchellh/go-wordwrap v1.0.0 h1:6GlHJ/LTGMrIJbwgdqdl2eEH8o+Exx/0m8ir9
 github.com/mitchellh/go-wordwrap v1.0.0/go.mod h1:ZXFpozHsX6DPmq2I0TCekCxypsnAUbP2oI0UX1GXzOo=
 github.com/mitchellh/mapstructure v0.0.0-20160808181253-ca63d7c062ee/go.mod h1:FVVH3fgwuzCH5S8UJGiWEs2h04kUh9fWfEaFds41c1Y=
 github.com/mitchellh/mapstructure v1.4.1/go.mod h1:bFUtVrKA4DC2yAKiSyO/QUcy7e+RRV2QTWOzhPopBRo=
+github.com/mitchellh/mapstructure v1.5.0/go.mod h1:bFUtVrKA4DC2yAKiSyO/QUcy7e+RRV2QTWOzhPopBRo=
 github.com/mmcloughlin/avo v0.5.0 h1:nAco9/aI9Lg2kiuROBY6BhCI/z0t5jEvJfjWbL8qXLU=
 github.com/mmcloughlin/avo v0.5.0/go.mod h1:ChHFdoV7ql95Wi7vuq2YT1bwCJqiWdZrQ1im3VujLYM=
 github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd h1:TRLaZ9cD/w8PVh93nsPXa1VrQ6jlwL5oN8l14QlcNfg=
@@ -1202,6 +1203,7 @@ github.com/shurcooL/go-goon v0.0.0-20170922171312-37c2f522c041 h1:llrF3Fs4018ePo
 github.com/spaolacci/murmur3 v0.0.0-20180118202830-f09979ecbc72 h1:qLC7fQah7D6K1B0ujays3HV9gkFtllcxhzImRR7ArPQ=
 github.com/spaolacci/murmur3 v0.0.0-20180118202830-f09979ecbc72/go.mod h1:JwIasOWyU6f++ZhiEuf87xNszmSA2myDM2Kzu9HwQUA=
 github.com/spf13/cast v1.6.0/go.mod h1:ancEpBxwJDODSW/UG4rDrAqiKolqNNh2DX3mk86cAdo=
+github.com/spf13/jwalterweatherman v1.1.0/go.mod h1:aNWZUN0dPAAO/Ljvb5BEdw96iTZ0EXowPYD95IqWIGo=
 github.com/stretchr/testify v1.6.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
 github.com/stretchr/testify v1.7.2/go.mod h1:R6va5+xMeoiuVRoj+gSkQ7d3FALtqAAGI1FQKckRals=
 github.com/stretchr/testify v1.7.5/go.mod h1:yNjHg4UonilssWZ8iaSj1OCr/vHnekPRkoO+kdMU+MU=
@@ -1279,7 +1281,6 @@ go.opentelemetry.io/proto/otlp v0.7.0 h1:rwOQPCuKAKmwGKq2aVNnYIibI6wnV7EvzgfTCzc
 go.opentelemetry.io/proto/otlp v0.7.0/go.mod h1:PqfVotwruBrMGOCsRd/89rSnXhoiJIqeYNgFYFoEGnI=
 go.opentelemetry.io/proto/otlp v1.0.0 h1:T0TX0tmXU8a3CbNXzEKGeU5mIVOdf0oykP+u2lIVU/I=
 go.opentelemetry.io/proto/otlp v1.0.0/go.mod h1:Sy6pihPLfYHkr3NkUbEhGHFhINUSI/v80hjKIs5JXpM=
-go.uber.org/atomic v1.11.0/go.mod h1:LUxbIzbOniOlMKjJjyPfpl4v+PKK2cNJn91OQbhoJI0=
 go.uber.org/multierr v1.6.0/go.mod h1:cdWPpRnG4AhwMwsgIHip0KRBQjJy5kYEpYjJxpXp9iU=
 go.uber.org/multierr v1.9.0/go.mod h1:X2jQV1h+kxSjClGpnseKVIxpmcjrj7MNnI0bnlfKTVQ=
 go.uber.org/zap v1.17.0/go.mod h1:MXVU+bhUf/A7Xi2HNOnopQOrmycQ5Ih87HtOu4q5SSo=
@@ -1652,6 +1653,7 @@ google.golang.org/protobuf v1.34.0/go.mod h1:c6P6GXX6sHbq/GpV6MGZEdwhWPcYBgnhAHh
 google.golang.org/protobuf v1.34.1/go.mod h1:c6P6GXX6sHbq/GpV6MGZEdwhWPcYBgnhAHhKbcUYpos=
 gopkg.in/alecthomas/kingpin.v2 v2.2.6 h1:jMFz6MfLP0/4fUyZle81rXUoxOBFi19VUFKVDOQfozc=
 gopkg.in/errgo.v2 v2.1.0 h1:0vLT13EuvQ0hNvakwLuFZ/jYrLp5F3kcWHXdRggjCE8=
+gopkg.in/ini.v1 v1.67.0/go.mod h1:pNLf8WUiyNEtQjuu5G5vTm06TEv9tsIgeAvK8hOrP4k=
 gopkg.in/yaml.v2 v2.2.3/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
 gopkg.in/yaml.v2 v2.2.8/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
 mvdan.cc/editorconfig v0.2.0 h1:XL+7ys6ls/RKrkUNFQvEwIvNHh+JKx8Mj1pUV5wQxQE=

